### DISCUSSION TITLE
[Feature Request]: Set Windows DPi scaling

### DISCUSSION THREAD

**Discussion by Marocco2** [ClassicOldSong/moonlight-android#32] [Ideas]
Hi!
I'm an advanced user of Sunshine/Moonlight and I setup a virtual display with custom resolution for my devices (65" TV, 27" TV, Smartphone, Laptop, iPad etc.) but my main issue remains the display scaling of these devices (I need to switch to 200% for small displays or when I play far away from 27" TV)
I kindly request if it's possible to make a user settings for (Windows) DPi settings without needing to do manually for every stream.
Thanks

**Comment by ClassicOldSong** [comment]
See here:
https://github.com/ClassicOldSong/Apollo/issues/60
